sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Azure.Core;
+
+namespace Azure.AI.Personalizer
+{
+    /// <summary> The Personalizer service client. </summary>
+    public partial class PersonalizerClient
+    {
+        /// <summary> The PersonalizerBase service client. </summary>
+        public PersonalizerBaseClient PersonalizerBase { get; set; }
+        /// <summary> The MultiSlot service client. </summary>
+        public MultiSlotClient MultiSlot { get; set; }
+        /// <summary> The MultiSlotEvents service client. </summary>
+        public MultiSlotEventsClient MultiSlotEvents { get; set; }
+        /// <summary> The Model service client. </summary>
+        public ModelClient Model { get; set; }
+        /// <summary> The Log service client. </summary>
+        public LogClient Log { get; set; }
+        /// <summary> The Events service client. </summary>
+        public EventsClient Events { get; set; }
+        /// <summary> The Evaluation service client. </summary>
+        public EvaluationClient Evaluation { get; set; }
+        /// <summary> The Evaluations service client. </summary>
+        public EvaluationsClient Evaluations { get; set; }
```

Comment:
Can these two be named more distinctly/descriptively so users know easily their purpose and usage.

Reply:
Evaluation Client was created because we have an operation ID which is Evaluation_Apply. The autorest takes the operation ID and creates the Evaluation client. Applying an evaluation is an action on the service configuration so it is more fitting to be under the ServiceConfiguration client. I will change the operation ID so that the evaluation client is not created and the apply evaluation is under the service configuration client. This will remove the Evaluation client and thus eliminate the need for renaming.